src/tasks/deploy.mk

```diff
@@ -93,15 +93,16 @@ deplo%: ## deploy: deploy the app to heroku
 
 change-api:
 	curl \
-  		--header "Content-Type: application/json" \
-  		--header "x-api-key: $(CHANGE_API_KEY)" \
+		--header "Content-Type: application/json" \
+		--header "x-api-key: $(CHANGE_API_KEY)" \
 		--request POST \
-  		--data "{ \
+		--data "{ \
 			\"user\": { \
 				\"githubName\": \"$(CIRCLE_USERNAME)\" \
 			}, \
 			\"environment\": \"production\", \
-			\"systemCode\": \"$(CIRCLE_PROJECT_REPONAME)\", \
+			\"systemCode\": \"$(shell curl https://next-registry.ft.com/v2/ | jq ".[] | select(.repository == \"https://github.com/${CIRCLE_USERNAME}/${CIRCLE_PROJECT_REPONAME}\") | .code")\", \
+			\"gitRepositoryName\": \"$(CIRCLE_USERNAME)/$(CIRCLE_PROJECT_REPONAME)\", \ 
```

Comment:
You want `CIRCLE_PROJECT_USERNAME`( the Bitbucket or Github Org name)  instead of `CIRCLE_USERNAME` ( the GH user initiating the build) according to the [docs](https://circleci.com/docs/2.0/env-vars/#built-in-environment-variables).

Reply:
Amazing spot!